ci-operator/jobs/openshift/installer/openshift-installer-presubmits.yaml

```diff
@@ -293,5 +293,7 @@ presubmits:
         - sh
         args:
         - ./hack/go-lint.sh
+        - cmd/...
```

Comment:
Why not `go list -f '{{ .Dir }}' ./...` ?

Reply:
> Why not `go list -f '{{ .Dir }}' ./...` ?

Something like that could work, although:

```console
$ sudo hack/go-lint.sh $(go list -f '{{ .Dir }}' ./...)
import "/home/trking/.local/lib/go/src/github.com/openshift/installer/cmd/openshift-install": cannot import absolute path
...
```

It works with `.ImportPath`:

```console
$ sudo hack/go-lint.sh $(go list -f '{{ .ImportPath }}' ./...)
```

How do I encode that in YAML here?  Something like:

```yaml
      - image: quay.io/coreos/golang-testing
        env:
        - name: IS_CONTAINER
          value: "TRUE"
        command:
        - sh
        args:
        - -c
        - ./hack/go-lint.sh $(go list -f '{{ .ImportPath }}' ./...)
```

Reply:
I suspect that PodSpec would work, yes

Reply:
> I suspect that PodSpec would work, yes

Updated with a69bc01 -> aaa039c.